app.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+import os
+
+from flask import Flask, request
+from ckip_transformers.nlp import CkipWordSegmenter, CkipPosTagger, CkipNerChunker
+
+
+class Config:
+  CKIP_DEVICE = os.environ.get('CKIP_DEVICE') or -1 
+  CKIP_TRANSFORMER_MODEL = os.environ.get('CKIP_TRANSFORMER_MODEL') or 'bert-base'
+
+app = Flask(__name__)
+app.config.from_object(Config())
```

Comment:
I don't think we have to setup flask config if the values are just for ourselves to use.
```suggestion
CONFIG = {
  'CKIP_DEVICE' :os.environ.get('CKIP_DEVICE') or -1 
  'CKIP_TRANSFORMER_MODEL': os.environ.get('CKIP_TRANSFORMER_MODEL') or 'bert-base'
}
```